Database/Corrections/TBC/tbcItemFixes.lua

```diff
@@ -103,6 +103,9 @@ function QuestieTBCItemFixes:Load()
         [30783] = {
             [itemKeys.objectDrops] = {184867},
         },
+        [32742] = {
+			[itemKeys.npcDrops] = {23363}
+		},
```

Comment:
Good catch on the `vendors` not being the `npcDrops` 👍🏻 

Could you fix the format of these two lines though?